.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,23 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: 'enhancement, priority: undecided'
```

Comment:
```suggestion
labels: 'feature, priority: undecided'
```
Why enhancement?

Reply:
It is the basic one from GitHub that is meant for features and refactoring

Reply:
Up to you